bump_rpm_packaging

```diff
@@ -9,16 +9,16 @@ cd "$PACKAGING_DIR"
 git fetch $PACKAGING_GIT_REMOTE
 git checkout -b rpm/release-$PROJECT-$FULLVERSION $PACKAGING_GIT_REMOTE/$REMOTE_BRANCH
 
-if [[ $FULLVERSION == *-RC* ]] ; then
-	if [[ FULLVERSION == *-RC1 ]] ; then
+if [[ $FULLVERSION == *-RC* ]] || [[ $FULLVERSION == *-rc* ]]; then
+	if [[ $FULLVERSION == *-RC1 ]] || [[ $FULLVERSION == *-rc1 ]]; then
```

Comment:
Bumping the release isn't needed when it's `rc1` (but is with `RC1`). Given we're switching to lowercase and don't expect any more uppercase, I think it's safe to drop this block